libs/core/src/lib/notification/notification-service/notification.service.ts

```diff
@@ -43,11 +49,11 @@ export class NotificationService {
         let notificationComponentRef: ComponentRef<NotificationComponent>;
 
         // Create Notification Component
-        notificationComponentRef = this.dynamicComponentService.createDynamicComponent(
+        notificationComponentRef = this._dynamicComponentService.createDynamicComponent(
             content,
             NotificationComponent,
             notificationConfig,
-            { services: [notificationService, notificationConfig] }
+            { services: [notificationService, notificationConfig, this._rtlService] }
```

Comment:
I suggest using the `injector` option instead.
`services` option is so unclear (not obvious what will be used as an injection token in this case). I would remove `services` option at all as the `injector` option is much more flexible and transparent

Reply:
Done. Please check